malware/system/ksp-block-linux-xorddos-trojan-process.yaml

```diff
@@ -0,0 +1,22 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-block-linux-xorddos-trojan-process
+  namespace: default
+spec:
+  tags: ["LINUX", "XORDDOS", "MALWARE"]
+  message: "Alert! Unusual binary [good] found on internal system directories! Possible Xorddos malware infection."
+  selector:
+    matchLabels:
+      container: xorddos
+  process:
+    severity: 1
+    process:
+    matchPaths:
+    - path: /bin/good
```

Comment:
This is good, if you can raise it separate PR
I can be able to merged, since this is an IoC